constrain/library/vav_turndown.py

```diff
@@ -0,0 +1,39 @@
+from constrain.checklib import RuleCheckBase
+
+REHEAT_DETERMINANT_TEMP = 5.56  # deg C ~= 10 deg F
+
+
+class VAVTurndown(RuleCheckBase):
+    points = [
+        "T_AHU_discharge",
+        "T_VAV_discharge",
```

Comment:
Could we perhaps make these be a reheat flag? That way the user can preprocess it. For the example that we were working off of we had these temperatures but perhaps in another situation a user might have the reheat coil output.

Reply:
Good point. Addressed.